<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu</class>
 <widget class="QWidget" name="Menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>600</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="styleSheet">
     <string notr="true">#page{border-image: url(:/new/prefix1/img/menubg1.png);}</string>
    </property>
    <widget class="QPushButton" name="local_game_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>180</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>本地游戏</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>440</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>退出游戏</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="about_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>关于项目</string>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="net_game_btn">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>310</y>
       <width>200</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>网络游戏</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true">#page_2{border-image: url(:/new/prefix1/img/menubg1.png);}</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>5</y>
       <width>550</width>
       <height>400</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>474</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>XXX棋王争霸赛</string>
     </property>
     <property name="maxLength">
      <number>64</number>
     </property>
     <property name="cursorPosition">
      <number>8</number>
     </property>
    </widget>
    <widget class="QLabel" name="connect_stat_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>418</y>
       <width>400</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="create_btn">
     <property name="geometry">
      <rect>
       <x>426</x>
       <y>474</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>创建房间</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh_btn">
     <property name="geometry">
      <rect>
       <x>426</x>
       <y>418</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>匹配</string>
     </property>
    </widget>
    <widget class="QLabel" name="room_name_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>474</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>房间标题：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_net_btn">
     <property name="geometry">
      <rect>
       <x>426</x>
       <y>530</y>
       <width>150</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>返回主页面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reconnect_btn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>400</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/img/btnbg.jpg);</string>
     </property>
     <property name="text">
      <string>连接服务器</string>
     </property>
    </widget>
    <widget class="QLabel" name="people_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>580</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
